workspace_host/interface.js

```diff
@@ -967,10 +968,22 @@ function _createContainer(workspace, callback) {
     } else {
         args = args.split(' ');
     }
+
+    let networkMode = 'bridge';
+    if (!workspace.settings.workspace_enable_networking) {
+        if (config.workspaceSupportNoInternet) {
+            networkMode = 'no-internet';
```

Comment:
Nit: it may be advantageous to make the name of the network configurable? Not necessary based on the current setup though.

Reply:
Skipping for now.